bin/local.js

```diff
@@ -68,7 +69,10 @@ function onFileChange(evt, filepath) {
 }
 
 function startFileWatch() {
-    watcher.watch(config.sourceLocation, {
+    if (fswatcher){
+        fswatcher.close();
```

Comment:
This is a good catch, but couldn't we do this instead:

```
if (!fswatcher) {
  fswatcher = ...
}
```
Seems a bit redundant to close and immediately re-open when we might as well just let the file watch persist?